test/gaudi_opts/test_edm4hep2lcio.py

```diff
@@ -0,0 +1,77 @@
+from Gaudi.Configuration import *
+
+from Configurables import k4DataSvc, MarlinProcessorWrapper, EDM4hep2LcioTool, k4LCIOReaderWrapper
+
+algList = []
+
+END_TAG = "END_TAG"
+
+# theFile= 'edminput.root'
+theFile = '/eos/experiment/fcc/ee/generation/DelphesEvents/fcc_tmp/p8_ee_Ztautau_ecm91_EvtGen_Tau2MuGamma/events_001720714.root'
```

Comment:
@clementhelsens Do you have  a file created with the latest delphes with the full and "transposed" covariance matrix"?

Reply:
from @vvolkl: put the file in an http accessible place